src/main/java/no/nav/fo/veilarbregistrering/oppgave/OppgaveService.java

```diff
@@ -98,4 +104,34 @@ public String getName() {
     protected LocalDate idag() {
         return LocalDate.now();
     }
+
+    /**
+     * Stegene som skal gjøres:
+     * 1) Hente en registrering som har feilet
+     * - avbryt hvis ingen funnet
+     * 2) Hent grunnlaget for oppgaven;
+     * - fødselsnummer (fra registreringen)
+     * 3) Opprette oppgave
+     * 4) Oppdatere status på registreringen
+     */
+    @Transactional
+    public void opprettOppgaveAsynk() {
+        // TODO Flytte til egen klasse?
```

Comment:
Ja - jeg tenker det er fint å flytte dette til egen klasse.

Reply:
Ref. tidligere kommentar om sirkulære avhengigheter - jeg kunne ha tenkt meg å flytte denne til registrering, og at den bruker oppgave for å lage oppgaven, men at den tar utgangspunkt i registreringssiden.